src/Altinn.Notifications.Integrations/Kafka/Consumers/AltinnServiceUpdateConsumer.cs

```diff
@@ -0,0 +1,57 @@
+﻿using Altinn.Notifications.Core.Models.AltinnServiceUpdate;
+using Altinn.Notifications.Core.Services.Interfaces;
+using Altinn.Notifications.Integrations.Configuration;
+
+using Microsoft.Extensions.Logging;
+
+using Microsoft.Extensions.Options;
+
+namespace Altinn.Notifications.Integrations.Kafka.Consumers
+{
+    /// <summary>
+    /// Kafka consumer class for Altinn service updates
+    /// </summary>
+    public class AltinnServiceUpdateConsumer : KafkaConsumerBase<AltinnServiceUpdateConsumer>
```

Comment:
I still believe each consumer type requires a unique consumer group id. It might be working for now because we have only one type of consumer per topic, and Kafka is probably not tracking the activity of a consumer group across topics.

Reply:
But will we ever get to the case where a single application consumes the same topic with different types of consumers? 
I guess we could alter the group Id to contains the name of the specific consumer T.